README.md

```diff
@@ -35,6 +35,33 @@ To facilitate upgrading in the future, removing and adding features, this is the
 - We can add feature flags to be able to test new functionality
 - Chromatic production can not remove any features this package depends on until after the usage has been removed from this package in addition to a grace period to allow users to upgrade
 
+### Building and running locally
+
+1. Ensure all dependencies are installed with `yarn install`
+2. Build + watch the code locally: `yarn dev`
+3. Run a build of all the CLI's stories against a Chromatic project: `yarn chromatic -t <token>`.
+
+#### Running against staging
+
+```bash
+CHROMATIC_INDEX_URL=https://www.staging-chromatic.com/graphql yarn chromatic -t 253df72b53d2
```

Comment:
```suggestion
CHROMATIC_INDEX_URL=https://index.staging-chromatic.com yarn chromatic -t 253df72b53d2
```

Reply:
Hmmmm.. I'm pretty sure you need the `/graphql` bit. Plus it is not really different (as www proxies to the index), but I find it simpler to just make it www.

PS I tested these commands they definitely work!

Reply:
I always run with `CHROMATIC_INDEX_URL=https://index.staging-chromatic.com` without the path 🤷‍♂️
Seems more sensible to me to set `INDEX_URL` to the actual index URL.

Reply:
TIL